Wbooks/Library/LibraryController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LibraryController.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 08/04/2021.
+//
+
+import Foundation
+import UIKit
+
+final class LibraryController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    // MARK: Properties
+    private lazy var libraryView: LibraryView = LibraryView()
+    
+    private let booksArray : [Book] = [Book(cover: UIImage(named: "BookCover1")!, title: "A Little Bird Told Me", author: "Timothy Cross"), Book(cover: UIImage(named: "BookCover2")!, title: "When the Doves Disappeared", author: "Sofi Oksanen"), Book(cover: UIImage(named: "BookCover3")!, title: "The Best Book in the World", author: "Peter Sjernstrom"), Book(cover: UIImage(named: "BookCover4")!, title: "Be Creative", author: "Tony Alcazar"), Book(cover: UIImage(named: "BookCover5")!, title: "Redesign the Web", author: "Liliana Castilla")]
```

Comment:
When you have such long lines, try to separate them between ","
example:
private let booksArray : [Book] = [Book(cover: UIImage(named: "BookCover1")!,
 title: "A Little Bird Told Me", 
 author: "Timothy Cross"),
...